hadoop-ozone/cli-shell/src/main/java/org/apache/hadoop/ozone/shell/acl/GetAclHandler.java

```diff
@@ -19,19 +19,61 @@
 
 import java.io.IOException;
 import java.util.List;
+import java.util.stream.Collectors;
 import org.apache.hadoop.ozone.OzoneAcl;
 import org.apache.hadoop.ozone.client.OzoneClient;
 import org.apache.hadoop.ozone.security.acl.OzoneObj;
+import picocli.CommandLine.Option;
 
 /**
  * Get ACLs.
  */
 public abstract class GetAclHandler extends AclHandler {
 
+  @Option(names = {"--string", "-o"},
```

Comment:
Thanks @echonesis  for working on this task. 

I would suggest add an following Option instead of this one,

```
  @CommandLine.Option(names = {"--json"},
      description = "Format output as JSON",
      defaultValue = "true")
  private boolean json;
```

Reply:
It's very good to thought that there are possible other formats to support in future. But the current Ozone CLI state is either json or plain output without structured format, so  --json is used in many CLIs. To have a consistent CLI option format, --json is preferred.  There is possibility to support xml or yaml in future, but we don't see such requirements so far. If it becomes true, then a lot of existing CLIs need to improved in batches. 

>  why have a flag for the default behavior?

A flag for the default behavior is not a problem. It provides a default behavior, and a option to change the default behavior.  Here --json with defaultValue="true"  is to keep the current command output same as without explicitly set the option.

Reply:
Thanks for the feedback! 
I want to confirm the backward compatibility impact: 
Current implementation (before this PR):
```
Default: JSON output
No --json flag option available
```

If we change it to be (defaultValue = "false"):
```
Default: String output
--json: JSON output
```
Since this command already exists in production and defaults to JSON output, changing to defaultValue = "false" would be a breaking change. 
- Should I keep defaultValue = "true" to maintain backward compatibility?
- Accept the breaking change and use defaultValue = "false" for consistency with other Ozone CLI commands?

Reply:
Thanks @adoroszlai for pointing this out.
I will add this feature in the next commit.